Sources/KlaviyoSwift/StateManagement.swift

```diff
@@ -228,8 +228,8 @@ struct KlaviyoReducer: ReducerProtocol {
             }
             switch networkStatus {
             case .notReachable:
-                state.flushInterval = 0
-                return EffectPublisher.cancel(ids: [RequestId.self, Timer.self])
+                state.flushInterval = Double.infinity
```

Comment:
I read this as double infinity at first... Like infinity wasn't enough, double it.

Reply:
😂